var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -307,6 +307,10 @@ class Hdf5(CMakePackage):
     )
 
     patch("fortran-kinds.patch", when="@1.10.7")
+    patch("fortran-kinds-2.patch", when="@1.12.0")
+    patch("mpi.patch", when="@1.12")
+    patch("mpi-perf.patch", when="@1.12.2")
+    patch("mpi.14.patch", when="@1.14:")
```

Comment:
Can we add a comment, above each patch directive, to explain briefly what the patch does , why it is needed and where it was taken from?

Reply:
@eschnett